apps/nrm/translations/src/en/pages.json

```diff
@@ -328,6 +328,13 @@
     "header": "Enter the email to send the Pdf"
   },
   "save-report": {
```

Comment:
This is all fine for now, but the content could be better; considering filing a ticket for a content designer to take a look at it eventually.